src/analysis/normed_space/normed_group_hom.lean

```diff
@@ -434,6 +434,17 @@ by { erw f.to_add_monoid_hom.mem_ker, refl }
   (incl g.ker).comp (ker.lift f g h) = f :=
 by { ext, refl }
 
+@[simp]
+lemma ker_zero : (0 : normed_group_hom V₁ V₂).ker = ⊤ :=
+by { ext, simp [normed_group_hom.mem_ker] }
+
+lemma ker_eq_preimage : (f.ker : set V₁) = (f : V₁ → V₂) ⁻¹' {0} :=
+by { ext, erw f.mem_ker }
```

Comment:
```suggestion
lemma coe_ker : (f.ker : set V₁) = (f : V₁ → V₂) ⁻¹' {0} :=
by { ext, erw f.mem_ker }
```
makes this consistent with `submonoid.coe_comap`.

Can you add this for `add_monoid_hom.ker` too?